delivery_carrier_label_postlogistics/postlogistics/web_service.py

```diff
@@ -44,12 +61,29 @@ def __init__(self, company):
         self.init_connection(company)
 
     def init_connection(self, company):
-        t = HttpAuthenticated(
-            username=company.postlogistics_username,
-            password=company.postlogistics_password)
-        self.client = Client(
-            company.postlogistics_wsdl_url,
-            transport=t)
+        if company.postlogistics_test_mode:
+            # We must change location for namespace as suds will take the wrong
+            # one and find nothing
+            # In order to have the test_mode working, it is necessary to patch
+            # suds with https://fedorahosted.org/suds/attachment/ticket/239/suds_recursion.patch # noqa
```

Comment:
Do you know if this was merged in the suds-jurko fork? If not maybe the patch could be proposed there?